Pod/Classes/CDTQQueryValidator.m

```diff
@@ -287,28 +349,46 @@ + (BOOL)validateClause:(NSDictionary *)clause
     //$exits lt
 
     NSArray *validOperators =
-        @[ @"$eq", @"$lt", @"$gt", @"$exists", @"$not", @"$ne", @"$gte", @"$lte" ];
+        @[ @"$eq", @"$lt", @"$gt", @"$exists", @"$not", @"$gte", @"$lte", @"$in" ];
```

Comment:
I stared at this comment for quite a while trying to make sense of it.  What a difference one letter makes!  I finally got it: "Maybe **NOTE** that things like `$ne` are normalized...".  

Anyway, I added a comment here to clarify why operators like `$ne` and `$nin` are not in the `validOperators` list.  Done in b667c92d0cf9994ca7a8751af6e7efedcb6f3199